### PR TITLE
BASK6: Prototype the Home Page

### PR DESCRIPTION
**Pull request by cvu1998**
BASK6 (Issue #9):
-Added tabs for Home, Profile
-Added Logout and Reload button
-Added search bar
-Added posts/tweets with their Like, Retweet, and Comment button

### CODE REVIEW

**Comment by Pwnagebird** [comment]
Rename branch to the issue name and add BASK6 (Issue #) to your commit message

**Comment by cvu1998** [comment]
Done

> Rename branch to the issue name and add BASK6 (Issue #) to your commit message

Done

**Comment by Pwnagebird** [comment]
In the comment commit add the issue #

**Comment by cvu1998** [comment]
> In the comment commit add the issue #

Done

File: src/main/java/com/basilisk/frontend/views/HomePageView.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    private Tab profile;
+    private Tabs tabs;
+
+    private Button logout;
+    private Button reload;
+
+    private HorizontalLayout searchBar;
+    private TextField searchTxt;
+    private Button searchButton;
+
```
Comment (Pwnagebird) [inline src/main/java/com/basilisk/frontend/views/HomePageView.java]: Put this method private

Comment (cvu1998) [reply src/main/java/com/basilisk/frontend/views/HomePageView.java]: Done

Comment (Pwnagebird) [reply src/main/java/com/basilisk/frontend/views/HomePageView.java]: Good to merge!

**Pull request merged**